proto/ibc/core/client/v1/tx.proto

```diff
@@ -90,6 +94,28 @@ message MsgUpgradeClient {
 // MsgUpgradeClientResponse defines the Msg/UpgradeClient response type.
 message MsgUpgradeClientResponse {}
 
+// MsgIBCSoftwareUpgrade defines an sdk.Msg to schedule an upgrade of an IBC client using a v1 governance proposal
+message MsgIBCSoftwareUpgrade {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the address of the governance account.
+  string                      authority = 1;
```

Comment:
Based on the discussion we had yesterday in the engineering call, should we then also rename this to `signer`?